ApollosProject/apollos-apps
# Move and refactor MediaControls

## DESCRIPTION
<img src="https://user-images.githubusercontent.com/2528817/73697939-78e45700-46a5-11ea-87ff-19a5b2c49b54.png" width="200"/><img src="https://user-images.githubusercontent.com/2528817/73697988-92859e80-46a5-11ea-9cb6-81e72d8759c7.png" width="200"/>

### What does this PR do, or why is it needed?
Moves the MediaControls components to the ui-connected package. It also breaks the single component down into few smaller components to make things a little more maintainable and testable.

### How do I test this PR?

* Open app
* Click on any Video(live and regular) and it should work as expected.

### What automated tests did you write?

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
